frontend/app/route-helpers/renew-route-helpers.server.ts

```diff
@@ -30,6 +30,7 @@ export interface RenewState {
     email?: string;
   };
   readonly hasAddressChanged?: boolean;
+  readonly areMailingAndHomeAddressSame?: boolean;
```

Comment:
Rename the property to a more descriptive name that accurately reflects its usage context. Refactor the codebase to replace all occurrences of the old name with the new name.

```suggestion
  readonly isHomeAddressSameAsMailingAddress?: boolean;
```